Detectors/ITSMFT/ITS/simulation/include/ITSSimulation/Detector.h

```diff
@@ -306,7 +308,7 @@ class Detector : public o2::Base::Detector
     GeometryHandler *mGeometryHandler;
     MisalignmentParameter *mMisalignmentParameter;
 
-    V1Layer **mGeometry;   //! Geometry
+    V3Layer **mGeometry;   //! Geometry
```

Comment:
I apologize for my lack of experience: could you please suggest me what I should improve here ?